recipe/meta.yaml

```diff
@@ -21,10 +20,9 @@ source:
     - patches/0007-CMake-VC9-compatibility-ignore-some-warnings.patch
     - patches/0008-VC9-compatibility-remove-C99.patch
     - patches/0009-CMake-Force-Multi-threaded-DLL-runtime.patch
-    - patches/0010-Use-CreateSymbolicLinkW-on-Windows-for-symlinks-WIP.patch
 
 build:
-  number: 1008
+  number: 0
   skip: true  # [win and vc<14]
   run_exports:
     # https://abi-laboratory.pro/index.php?view=timeline&l=libarchive
```

Comment:
currently for some weird reason host contains `libiconv` for OS X and Linux, while run only has it for `OS X`. 
I think that's why I had to start adding `libiconv` to mamba at some point. Maybe we can make this consistent?

Reply:
@wolfv and @mingwandroid I removed `libiconv` from Linux builds. Hope that won't trigger issues downstream :smile: 

This is ready for a review!